src/oss/schemas/ReplicationCreationRequest.yml

```diff
@@ -0,0 +1,26 @@
+type: object
+properties:
+  name:
+    type: string
+  description:
+    type: string
+  orgID:
+    type: string
+  remoteID:
+    type: string
+  localBucketID:
+    type: string
+  remoteBucketID:
+    type: string
+  maxQueueSizeBytes:
+    type: int
+    format: int64
+    min: 33554430 # 32 MiB
+    default: 67108860 # 64 MiB
+required:
```

Comment:
Similar question for the "skip-verify/insecure TLS" on the `Replication` object.

Reply:
Same as above, captured in the referenced `remoteID`